### PR TITLE
Codacy

### PR DESCRIPTION
**Pull request by zuphilip**
(splitted out from #64)

We first should merge #64 and #66 and then this PR.

### CODE REVIEW

**Comment by stweil** [comment]
Thanks. The PR currently has some merge conflicts. I therefore merged the 4th commit manually. As the first two commits were already merged, there only remains a single commit.

**Comment by zuphilip** [comment]
Okay, just give me a minute. I will rebase the missing commit here...

**Comment by zuphilip** [comment]
🏓

**Comment by kba** [comment]
I prefer single quotes when possible because you don't need to escape double slashes in strings then.

I also prefer `let` over `var` because it scopes variables to block.

Semicolons are unnecessary except in very few circumstanceds (before calling methods on literal array expressions e.g.) I skip them nowadays whenever possible.

If you set these rules up with eslint, you can run `eslint --fix` to automatically enforce them.

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM alpine:edge
 MAINTAINER Konstantin Baierer <konstantin.baierer@gmail.com>
 
 EXPOSE 8080
-ADD . /ocr-fileformat
+COPY . /ocr-fileformat
```
Comment (stweil) [inline Dockerfile]: Looks good. I suggest to create a separate PR for this change, so it can be merged.

**Comment by zuphilip** [comment]
In this PR I tried to focus on improving Codacy result. My goal is to become a better grade. I looked at the present JS-code and from there choose the standards to follow further. We can look whether we can easily agree on coding rules or whether we should postpone some of these decisions.

> Single quotes vs. double quotes

Both are fine for me, but making them more consistently should be the goal.

> let vs. var

We can IMO also adjust some `var`s with `let`s.

> semicolons

I am used to add them and would prefer to continue adding them.

> eslint

This sounds interesting. Would this be locally?

**Comment by zuphilip** [comment]
@kba I would be fine if you create an alternative PR with `eslint` (but add the semicolons if you want to do me a favor).

**Comment by zuphilip** [comment]
Superseeded by #69 and the Dockerfile change is already pushed thanks to @kba.

**Pull request closed without merging**